bamboo_twig_loader/src/TwigExtension/Loader.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Drupal\bamboo_twig_loader\TwigExtension;
+
+use Drupal\bamboo_twig\TwigExtension\TwigExtensionBase;
+
+/**
+ * Provides some loaders as Twig Extensions.
+ */
+class Loader extends TwigExtensionBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFunctions() {
+    return [
+      new \Twig_SimpleFunction('bamboo_load_entity', [$this, 'loadEntity']),
+      new \Twig_SimpleFunction('bamboo_load_field', [$this, 'loadField']),
+      new \Twig_SimpleFunction('bamboo_load_currentuser', [$this, 'loadCurrentUser']),
+      new \Twig_SimpleFunction('bamboo_load_image', [$this, 'loadImage']),
```

Comment:
Create a issue for this here : https://github.com/antistatique/drupal-bamboo-twig/issues/7
People will chose.